stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -133,23 +157,74 @@ class PaymentSheet internal constructor(
         }
     }
 
+    /**
+     * A class that presents the individual steps of a payment sheet flow.
+     */
     internal interface FlowController {
-        fun getPaymentOption(): PaymentOption?
 
-        fun configure(
+        /**
+         * Configure the FlowController to process a [PaymentIntent].
+         *
+         * @param[paymentIntentClientSecret] the client secret for the [PaymentIntent].
+         * @param[configuration] optional [PaymentSheet] settings.
+         * @param[callback] called with the result of configuring the FlowController.
+         */
+        fun configureWithPaymentIntent(
             paymentIntentClientSecret: String,
-            configuration: Configuration,
+            configuration: Configuration? = null,
             callback: ConfigCallback
         )
 
-        fun configure(
+        /**
+         * @see [configureWithPaymentIntent] above.
+         */
+        fun configureWithPaymentIntent(
             paymentIntentClientSecret: String,
             callback: ConfigCallback
+        ) {
+            configureWithPaymentIntent(paymentIntentClientSecret, null, callback)
+        }
+
+        /**
+         * Configure the FlowController to process a [SetupIntent].
+         *
+         * @param[setupIntentClientSecret] the client secret for the [SetupIntent].
+         * @param[configuration] optional [PaymentSheet] settings.
+         * @param[callback] called with the result of configuring the FlowController.
+         */
+        fun configureWithSetupIntent(
+            setupIntentClientSecret: String,
+            configuration: Configuration? = null,
+            callback: ConfigCallback
         )
 
+        /**
+         * @see [configureWithSetupIntent] above.
```

Comment:
```suggestion
         * @see [configureWithSetupIntent].
```

Reply:
I added that because it's not well defined where the link will point too, as Dokka doesn't support overloaded methods yet https://github.com/Kotlin/dokka/issues/80